src/platforms/native/guides/wasm/index.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: WebAssembly
+description: "Learn about how Sentry processes WASM crash reports."
+---
+
+WebAssembly allows you to compile native code into a platform-independent bytecode format, which, for example, permits you to run native C, C++, or Rust code in a web browser.
+
+**This feature is still developing, so may be rough around the edges.**
+
+Debugging support for WebAssembly is an ongoing effort by the WebAssembly community.
+Support for it in Sentry is limited by what's possible in that ecosystem.
+
+<Note>
+
+Want to be an early adopter, but running into issues? Please reach out to
+us using our [GitHub Forum](https://forum.sentry.io/), we would love to talk.
```

Comment:
Thats not really a "github forum", is it?

Reply:
indeed.